developers_chamber/ecs_utils.py

```diff
@@ -559,12 +575,18 @@ def start_service_and_wait_for_tasks_to_start(cluster, service, count, region):
 def get_min_capacity_for_service(cluster, service, region):
     as_client = _get_autoscaling_client(region)
 
-    response = as_client.describe_scalable_targets(
-        ServiceNamespace='ecs',
-        ResourceIds=[
-            'service/{}/{}'.format(cluster, service),
-        ],
-    )
+    try:
+        response = as_client.describe_scalable_targets(
+            ServiceNamespace='ecs',
+            ResourceIds=[
+                'service/{}/{}'.format(cluster, service),
+            ],
+        )
+    except ecs_client.exceptions.ServiceNotFoundException:
+        raise ClickException("Service not found: '{}'".format(service))
```

Comment:
Říkám si, jestli bychom tady měli vyhazovat opravdu `ClickException` a ne nějakou naši interní výjimku. Pak tam totiž při volání této funkce vzniká obdobný kód jako níže, kdy se odchytne jedna `ClickException` jen proto, aby se vyhodila další a není z toho úplně jasné, proč se to vlastně stalo. Viděl bych tu nějakou base `PydevException` a z ní třeba `ServiceCapacityError` se zprávou. 
```python
try:
    count_per_service = get_min_capacity_for_service(cluster=cluster, service=service_name, region=region)
except ClickException as ex:
    raise ClickException(
        'Set explicit count or set up autoscaling with minimum capacity.\n{}'.format(ex)
    )
```
Tohle může být na více místech, všiml jsem si toho ale až tady.

Reply:
jo vím o tom, to be implemented

problém je že se některé funkce volají jak z kódu tak z CLI